pkg/cluster/config.go

```diff
@@ -5,22 +5,28 @@ import (
 )
 
 type Config struct {
-	Cluster               *anywherev1.Cluster
-	VSphereDatacenter     *anywherev1.VSphereDatacenterConfig
-	DockerDatacenter      *anywherev1.DockerDatacenterConfig
-	SnowDatacenter        *anywherev1.SnowDatacenterConfig
-	VSphereMachineConfigs map[string]*anywherev1.VSphereMachineConfig
-	SnowMachineConfigs    map[string]*anywherev1.SnowMachineConfig
-	OIDCConfigs           map[string]*anywherev1.OIDCConfig
-	AWSIAMConfigs         map[string]*anywherev1.AWSIamConfig
-	GitOpsConfig          *anywherev1.GitOpsConfig
-	FluxConfig            *anywherev1.FluxConfig
+	Cluster                  *anywherev1.Cluster
+	CloudStackDatacenter     *anywherev1.CloudStackDatacenterConfig
```

Comment:
can we add unit test in `parse_test.go`?

Reply:
done